### PR TITLE
Effect data arguments updates

### PR DESCRIPTION
**Pull request by Drummerboy444**

### CODE REVIEW

File: scripts/DashEffectData_GetDashMultiplyer/DashEffectData_GetDashMultiplyer.gml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+/// @function DashEffectData_GetDashMultiplyer(DashEffectData_data) Returns the dash multiplyer.
```
Comment (nick-wilson95) [inline scripts/DashEffectData_GetDashMultiplyer/DashEffectData_GetDashMultiplyer.gml]: Multiplier

File: scripts/DashEffectData_GetSlowMultiplyer/DashEffectData_GetSlowMultiplyer.gml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+/// @function DashEffectData_GetSlowMultiplyer(DashEffectData_data) Returns the slow multiplyer.
```
Comment (nick-wilson95) [inline scripts/DashEffectData_GetSlowMultiplyer/DashEffectData_GetSlowMultiplyer.gml]: Multiplier

File: scripts/DashEffectData_New/DashEffectData_New.gml [hunk 3]
```diff
@@ -0,0 +1,21 @@
+/// @function DashEffectData_New(num_steps, id_caster, num_dashMultiplyer, num_slowMultiplyer, num_slowSteps)
+///		Creates a new dash effect data.
+/// @param {number} num_steps The number of steps.
+/// @param {id} id_caster The instance id of the caster.
+/// @param {number} num_dashMultiplyer The multiplicative effect of the dash on the targets speed.
+/// @param {number} num_slowMultiplyer The multiplicative effect of the subsequent slow on the targets speed.
+/// @param {number} num_slowSteps The number of steps for the subsequent slow.
+/// @returns {DOTEffectData} The new slow effect data.
```
Comment (nick-wilson95) [inline scripts/DashEffectData_New/DashEffectData_New.gml]: So making a new DashEffectData returns "The new slow effect data", which has type DOTEffectData?

File: scripts/SlowEffectData_GetSlowMultiplyer/SlowEffectData_GetSlowMultiplyer.gml [hunk 4]
```diff
@@ -0,0 +1,12 @@
+/// @function SlowEffectData_GetSlowMultiplyer(SlowEffectData_data) Returns the slow multiplyer.
```
Comment (nick-wilson95) [inline scripts/SlowEffectData_GetSlowMultiplyer/SlowEffectData_GetSlowMultiplyer.gml]: Multiplier

**Comment by nick-wilson95** [comment]
[Updating the arguments array for effects](https://trello.com/c/rpyKF9hB/5-updating-the-arguments-array-for-effects)

File: scripts/DashEffectData_New/DashEffectData_New.gml (hunk 3 above)
Comment (Drummerboy444) [reply scripts/DashEffectData_New/DashEffectData_New.gml]: Oh boy, that's a pretty bad cumulative error of copy and paste haha

**Pull request merged**